code/modules/modular_computers/file_system/programs/generic/ntnrc_client.dm

```diff
@@ -156,6 +166,28 @@
 			channel.password = ""
 		else
 			channel.password = newpassword
+	if(href_list["PRG_directmessage"])
+		. = TRUE
+		var/clients = list()
+		var/names = list()
+		for(var/cl in ntnet_global.chat_clients)
+			var/datum/computer_file/program/chatclient/C = cl
+			clients[C.username] = C
+			names += C.username
+		var/picked = input(usr, "Select with whom you would like to start a conversation.") in names
```

Comment:
make null an option so you can exit out of it if the person you want to chat with isn't active